packages/web3torrent/src/index.tsx

```diff
@@ -1,9 +1,16 @@
+// TODO: Replace with injection via other means than direct app import
+// NOTE: This adds `channelProvider` to the `Window` object
 import '@statechannels/channel-provider';
+
 import React from 'react';
 import ReactDOM from 'react-dom';
 import App from './App';
 import {web3torrent, WebTorrentContext} from './clients/web3torrent-client';
 
+// TODO: Put inside better place than here where app can handle error case
```

Comment:
Perhaps we want to take our cues from RPS, bearing in mind that we need to get addresses from the wallet and that this can trigger the `window.ethereum.enable()` call.